src/RequirementsQuery.ts

```diff
@@ -0,0 +1,141 @@
+import { Calculator } from 'fqm-execution';
+import { R4 } from '@ahryman40k/ts-fhir-types';
+import fs from 'fs';
+import axios from 'axios';
+import { URLSearchParams } from 'url';
+import { DRQuery, APIParams, BulkDataResponse } from './types/RequirementsQueryTypes';
+
+const headers = {
+  Accept: 'application/fhir+json',
+  Prefer: 'respond-async'
+};
+const exampleMeasureBundle = '../EXM130-7.3.000-bundle.json'; //REPLACE WITH PATH TO DESIRED MEASURE BUNDLE
+const API_URL =
```

Comment:
Eventually this probably will want to be configurable. But that could wait until we add a `cli` Typescript file if we want.